e2e_test/test_fee_currency_fails_intrinsic.sh

```diff
@@ -7,6 +7,7 @@ source debug-fee-currency/lib.sh
 
 # Expect that the creditGasFees failed and is logged by geth
 tail -F -n 0 geth.log >debug-fee-currency/geth.intrinsic.log & # start log capture
+# trap 'kill %%' EXIT # kill bg job at exit
```

Comment:
If this works, we should use it. If not, I would like to know why are either add that as a comment or remove this line.

Reply:
I was fiddling around with this for a while couldn't understand why the test was hanging. In fact the hang was caused by `send_tx.mjs` not exiting (fixed by calling `process.exit(0)`). But also if the test was interrupted you could end up with multiple tails running in the background, which is what brought me to this.

I've re-instated it and removed the other kill command.